app/process/watcher/crypto-transfer/crypto-transfer.go

```diff
@@ -1,3 +1,61 @@
 package crypto_transfer
 
-// crypto transfer watcher
+import (
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	http "github.com/limechain/hedera-eth-bridge-validator/app/clients/http"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/crypto-transfer-message"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/proceed"
+	"github.com/limechain/hedera-watcher-sdk/queue"
+	"log"
+	"strconv"
+	"time"
+)
+
+type CryptoTransferWatcher struct {
+	client      *http.Client
+	accountID   hederasdk.AccountID
+	typeMessage string
+}
+
+func (ctw CryptoTransferWatcher) Watch(queue *queue.Queue) {
+	go ctw.beginWatching(ctw.accountID, ctw.typeMessage, queue)
+}
+
+func (ctw CryptoTransferWatcher) beginWatching(account hederasdk.AccountID, typeMessage string, q *queue.Queue) {
+	lastObservedTimestamp := strconv.FormatInt(time.Now().Unix(), 10)
+	for {
+		transactions, e := ctw.client.GetTransactionsByAccountIdAndTimestamp(account, lastObservedTimestamp)
+		if e != nil {
+			log.Printf("Suddenly stopped monitoring config [%s]\n", account.String())
+			log.Println(e)
+			return
+		}
+
+		if len(transactions.Transactions) > 0 {
+			log.Printf("After [%s] - Account [%s] - Transactions Length: [%d]\n", lastObservedTimestamp, account.String(), len(transactions.Transactions))
+			for _, tx := range transactions.Transactions {
+				log.Printf("[%s] - New transaction on config [%s] - Tx Hash: [%s]\n",
+					tx.ConsensusTimestamp,
+					account.String(),
+					tx.TransactionHash)
+
+				information := crypto_transfer_message.CryptoTransferMessage{
+					TxMemo: tx.MemoBase64,
+					Sender: tx.Transfers[len(tx.Transfers)-2].Account,
+					Amount: tx.Transfers[len(tx.Transfers)-1].Amount,
+				}
+				proceed.Proceed(information, typeMessage, account, q)
+			}
+			lastObservedTimestamp = transactions.Transactions[len(transactions.Transactions)-1].ConsensusTimestamp
+		}
+		time.Sleep(5 * time.Second)
```

Comment:
I think that the frequency of this call should be extracted into a configuration parameter. We can discuss it.